frontend/src/reducers/plannerSlice.ts

```diff
@@ -326,16 +326,24 @@ const plannerSlice = createSlice({
       // For now, keep courses hidden if summer column is also hidden. If we want
       // to remove the courses if summer term is toggled off, then 'term complete'
       // must also be reset.
-      // if (!state.isSummerEnabled) {
-      //   for (let i = 0; i < state.numYears; i++) {
-      //     const courses = state.years[i].T0;
-      //     courses.forEach((course) => {
-      //       state.courses[course].plannedFor = null;
-      //       state.unplanned.push(course);
-      //     });
-      //     state.years[i].T0 = [];
-      //   }
-      // }
+      if (!state.isSummerEnabled) {
+        for (let i = 0; i < state.numYears; i++) {
+          const courses = state.years[i].T0;
+          courses.forEach((course) => {
+            if (state.courses[course].isMultiterm) {
```

Comment:
if its multiterm. wouldnt we want to just push the offering down to the next spot?

Reply:
that was my choice - feel like it would be less confusing if we just removed every instance rather than shifting the multiterm instaces to the next term.